Remotion/Core/Reflection.UnitTests/MethodInfoAdapterTest.cs

```diff
@@ -60,6 +60,18 @@ public void Create_ReturnsSameInstance ()
       Assert.That (MethodInfoAdapter.Create (_method), Is.SameAs (MethodInfoAdapter.Create (_method)));
     }
 
+    [Test]
+    public void CreateOrNull_ReturnsSameInstance ()
+    {
+      Assert.That (MethodInfoAdapter.CreateOrNull (_method), Is.SameAs (MethodInfoAdapter.Create (_method)));
+    }
+
+    [Test]
+    public void CreateOrNull_ReturnsNull_ForNullMethodInfo ()

```

Comment:
naming: CreateOrNull_WithMethodInfoNull_ReturnsNull

Reply:
Are all new tests written using this pattern? I tried to adapt it to existing test names in the fixture.

Reply:
yes, unless there's a name bug.
it's always methodname (when there's fixture per type), then the context, and the result/exception comes last. sometimes we skip the result, but we don't mix the ordering.